packages/workspace/src/generators/move/schema.json

```diff
@@ -7,15 +7,18 @@
   "type": "object",
   "examples": [
     {
-      "command": "nx g @nrwl/workspace:move --project my-feature-lib shared/my-feature-lib",
+      "command": "nx g @nrwl/workspace:move --project my-feature-lib --destination shared/my-feature-lib",
       "description": "Move `libs/my-feature-lib` to `libs/shared/my-feature-lib`"
     }
   ],
   "properties": {
     "projectName": {
       "type": "string",
       "alias": "project",
-      "description": "The name of the project to move."
+      "description": "The name of the project to move.",
+      "$default": {
+        "$source": "projectName"
+      }
```

Comment:
I added this for auto complete in nx console, but I _kind of_ want to make it positional args as well so it can be similar to the unix `mv` command 
`nx g @nrwl/workspace:move <source-project> <path-to-destination` 

but I don't think we allow for 2 kinds of auto complete positional args and nx console completions?

Reply:
You should be able to use the `x-dropdown` field which Jon added to Nx Console. Like

```
"name": {
  "type": "string",
  "description": "bla bla",
  "$default": {
	"$source": "argv",
	"index": 0
  },
  "x-dropdown": "projects"
},
```

Reply:
that worked 🎉